benchmark/buffers/buffer-write.js

```diff
@@ -67,12 +77,23 @@ function main({ n, buf, type }) {
 
   if (!/\d/.test(fn))
     benchSpecialInt(buff, fn, n);
+  else if (/BigU?Int/.test(fn))
+    benchBigInt(buff, fn, BigInt(n));
   else if (/Int/.test(fn))
     benchInt(buff, fn, n);
   else
     benchFloat(buff, fn, n);
 }
 
+function benchBigInt(buff, fn, n) {
+  const m = mod[fn];
+  bench.start();
+  for (var i = 0n; i !== n; i++) {
+    buff[fn](i & m, 0);
+  }
+  bench.end(Number(n));
```

Comment:
It's minor, but I had trouble choosing between the following options:

1. `n` and `i` are Numbers, convert `i` to BigInt in the call (`BigInt(i) & m`).
2. `n` is Number, `i` is BigInt, use `!=` in the loop.
3. `n` and `i` are BigInts, convert to Number before `bench.end()` call.

It's now using option 3, but I'm not sure it's the best.

Reply:
It's already converted to BigInt in `main()`.

Reply:
I mean the coercion in `bench.end(Number(n))`.